global/global_player.lua

```diff
@@ -336,7 +336,7 @@ function event_level_up(e)
     eq.popup("", "<c \"#F0F000\">Welcome to level 5.</c><br><br>You have just been granted a new ability called '<c \"#F0F000\">Origin</c>' which allows you to teleport back to your starting city.<br><br>Open the Alternate Advancement window by pressing the '<c \"#F0F000\">V</c>' key, look in the '<c \"#F0F000\">General' tab</c>, and find the '<c \"#F0F000\">Origin</c>' ability and select it.<br><br>Now press the '<c \"#F0F000\">Hotkey</c>' button to create a hotkey you can place on your hot bar.");
   end
 
-  if e.self:GetLevel() == 10 then
+  if e.self:GetLevel() == 10 and eq.is_the_planes_of_power_enabled() then
```

Comment:
PoK armor quests released with the updated Tutorial June 29th 2005 - http://www.tski.co.jp/baldio/patch/20050629.html

So this should be Dragons of Norrath flagged

Reply:
Changed. Let me know if you see anything else!